petab/visualize/plot_residuals.py

```diff
@@ -100,3 +101,81 @@ def plot_residuals_vs_simulation(
 
     plt.tight_layout()
     return ax
+
+
+def plot_goodness_of_fit(
+        petab_problem: Problem,
+        simulations_df: Union[str, Path, pd.DataFrame],
+        size: Tuple = (10, 7),
+        ax: Optional[plt.Axes] = None
+) -> matplotlib.axes.Axes:
+    """
+    Plot goodness of fit.
+
+    Parameters
+    ----------
+    petab_problem:
+        A PEtab problem.
+    simulations_df:
+        A simulation DataFrame in the PEtab format or path to the simulation
+        output data file.
+    size:
+        Figure size.
+    ax:
+        Axis object.
+
+    Returns
+    -------
+        ax: Axis object of the created plot.
+    """
+
+    if isinstance(simulations_df, (str, Path)):
+        simulations_df = get_simulation_df(simulations_df)
+
+    if simulations_df is None or petab_problem.measurement_df is None:
+        raise NotImplementedError('Both measurements and simulation data '
+                                  'are needed for goodness_of_fit')
+
+    residual_df = calculate_residuals(
+        measurement_dfs=petab_problem.measurement_df,
+        simulation_dfs=simulations_df,
+        observable_dfs=petab_problem.observable_df,
+        parameter_dfs=petab_problem.parameter_df)[0]
+    slope, intercept, r_value, p_value, std_err = stats.linregress(
+        petab_problem.measurement_df['measurement'],
+        simulations_df['simulation'])  # x, y
```

Comment:
you calculate here the linear regression between measurement and simulation. What would  it tell me if this is not a diagonal plot?

Reply:
You could see here if simulated values tend to be larger or smaller that measurement values. This plot probably won't be useful in every case. Also it gives some measure of how much off the simulation is.  
It could be useful if one wants to compare different approaches to fit the model.